tools/export/mcuxpresso/empty_cproject.tmpl

```diff
@@ -0,0 +1,3 @@
+{% extends "mcuxpresso/.cproject.tmpl" %}
+
+{% block cpu_config %}{% endblock %}
```

Comment:
this is the common template, the target specific templates define only the target definition for the IDE. Without the target settings the debugger complains.